tests/stubs/logs/20220615.log

```diff
@@ -0,0 +1,4 @@
+[ 2022-06-15 10:24:43 ] 
+[ 2022-06-15 10:27:24 ] 
```

Comment:
Also: what are these data anyway? What's to purpose of this file?

Reply:
It was our interpretation of `It would be nice to store information about last build of new cache.` from #21

Reply:
About last build, not all of them. :)

Reply:
All right. If we would change code to store info about last bulid only, it would be ok? By 'it' I mean the way we put and store data now

Reply:
Maybe something like this would be better?
```json
{
  "lastBuild": "2022-06-15 10:24:43"
}
```

Then it would be nicely extendable. Try to use `json_encode` function here.